modules/ting_subsearch_common/ting_subsearch_common.module

```diff
@@ -0,0 +1,96 @@
+<?php
+
+/**
+ * @file
+ * Ting Subsearch Common.
+ */
+
+/**
+ * Do a secondary search juggling around the drupal static.
+ *
+ * @param string $keys
+ *   The search phrase.
+ *
+ * @return TingClientSearchResult
+ *   The search result of the secondary search.
+ */
+function ting_subsearch_common_do_secondary_search($keys, $options = array()) {
+
+  // Store search result temporarily. We need to make seconday search.
+  $temporary_search_result = drupal_static('ting_search_results');
+
+  // Reset the static containing search result to be able do new search.
+  drupal_static_reset('ting_search_search_execute');
+
+  // Load original variables to temporary before changing them.
+  if (!empty($options['profile'])) {
+    $temporary_profile = variable_get('ting_search_profile', '');
+    variable_set('ting_search_profile', $options['profile']);
+  }
+  if (!empty($options['agency'])) {
+    $temporary_agency = variable_get('ting_agency', FALSE);
+    variable_set('ting_agency', $options['agency']);
+  }
+
+  // Search for the secondary keys.
+  ting_search_search_execute($keys, array('sort' => ''));
+
+  $secondary_result = drupal_static('ting_search_results');
+
+  // Overwrite the static with the original search.
+  $revert = &drupal_static('ting_search_results');
+  $revert = $temporary_search_result;
+
+  if (!empty($temporary_profile)) {
+    variable_set('ting_search_profile', $temporary_profile);
+  }
+  if (!empty($temporary_agency)) {
+    variable_set('ting_agency', $temporary_agency);
+  }
+
+  return $secondary_result;
+}
+
+/**
+ * Implements hook_libraries_info().
+ */
+function ting_subsearch_common_libraries_info() {
+  return array(
+    'guzzle' => array(
+      'name' => 'Guzzle',
+      'vendor url' => 'https://github.com/guzzle/guzzle',
+      'download url' => 'https://github.com/guzzle/guzzle',
+      'version' => '6.2',
+      'xautoload' => function ($adapter) {
+        $adapter->composerJson('composer.json');
```

Comment:
The issues reported by Scrutinizer regarding indentation isen't in my opinion an issue.
Locally PHP_CodeSniffer reports this as OK. Perhaps Scrutinizer is running an older version of PHP_CodeSniffer ?